kitsune/questions/static/questions/js/stores/QuestionEditStore.es6.js

```diff
@@ -1,134 +0,0 @@
-/* globals _:false */
-
-import BaseStore from '../../../sumo/js/stores/BaseStore.es6.js';
```

Comment:
BaseStore is only imported in UrlStore, which isn't imported anywhere else, they can all be removed